spk/transmission/Makefile

```diff
@@ -11,15 +11,16 @@ DESCRIPTION_FRE = Transmission est un client BitTorrent simple et rapide. Vous p
 DESCRIPTION_SPN = Transmission es un cliente BitTorrent simple y rápido. Puedes controlarlo remotamente mediante su interfaz web o alguna aplicación dedicada.
 STARTABLE = yes
 DISPLAY_NAME = Transmission
-CHANGELOG = "1. Update openssl to 1.1.1n.<br/>2. Deploy dedicated package for DSM7 on comcerto2k (DS414j)."
+CHANGELOG = "1. Update Transmission to v4.0.2.<br/>2. Update openssl to 1.1.1t."
```

Comment:
I noticed openssl being remove from `DEPENDS` above in `cross/transmission/Makefile`

Reply:
You are correct. I do think that `openssl` is automatically pulled in with `curl` but I can remove it from the change log since it's not explicitly referenced.

Reply:
You are right, it is part of curl.  To me options are to add it as a dependency and therefore add a relevant comment, or nothing at all (others may see things differently).

Reply:
The change log has been updated to remove mention of `openssl`.